src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java

```diff
@@ -136,8 +136,9 @@ public CatalogObjectMetadataList create(
     @ApiResponses(value = { @ApiResponse(code = 404, message = "Bucket or catalog object not found"),
                             @ApiResponse(code = 401, message = "User not authenticated"),
                             @ApiResponse(code = 403, message = "Permission denied") })
+    @ResponseStatus(HttpStatus.OK)
     @RequestMapping(value = "/{name}", method = GET)
-    public ResponseEntity<CatalogObjectMetadata> get(
+    public CatalogObjectMetadata get(
```

Comment:
Can you explain to me the reason for returning `CatalogObjectMetadata` instead of `ResponseEntity<CatalogObjectMetadata>`. Is that design? Practical decision?

Reply:
Using ResponseEntity is more verbal and doesn't bring any values there. I think that using ResponseEntity should be done when returning several status without throwing exception or several type. However I think that this not a good practice. All endpoint should return a precise type and a related status. I think that ResponseEntity is useful to hide bad design decision when using rest that's why I push to avoid using it.